botcore/utils/members.py

```diff
@@ -0,0 +1,48 @@
+import typing
```

Comment:
This one is missing a module docstring

Reply:
https://github.com/python-discord/bot-core/pull/29/commits/94e7367baadb6229cf28530e6c39e06a37e3c9bd